script2.js

```diff
@@ -0,0 +1,170 @@
+let age = prompt("How old are you?");
+if (age >= 0 && age < 2) alert("You are a child");
+if (age >= 12 && age < 18) alert("You are a teenager");
+if (age >= 18 && age < 60) alert("You are an adult");
+if (age >= 60 && age < 100) alert("You are a pensioner");
+
+let num = prompt("Choose number from 0 to 9");
+switch (num) {
+  case "0":
+    alert(")");
+    break;
+  case "1":
+    alert("!");
+    break;
+  case "2":
+    alert("@");
+    break;
+  case "3":
+    alert("#");
+    break;
+  case "4":
+    alert("$");
+    break;
+  case "5":
+    alert("%");
+    break;
+  case "6":
+    alert("^");
+    break;
+  case "7":
+    alert("&");
+    break;
+  case "8":
+    alert("*");
+    break;
+  case "9":
+    alert("(");
+    break;
+}
+
+let number = prompt("Enter 3 digit number");
+number = number + "";
+if (
+  number[0] == number[1] ||
+  number[0] == number[2] ||
+  number[1] == number[2]
+) {
+  alert("TRUE");
+} else alert("FALSE");
+
+let year = prompt("Enter year you want to check");
+if (year % 400 == 0 || (year % 4 == 0 && year % 100 != 0)) {
+  alert("This is a leap year");
+} else {
+  alert("This is not a leap year");
+}
+
+let number = prompt("Enter 5 digit number");
+if (number[0] == number[4] || number[1] == number[3]) {
+  alert("This number is a palindrome");
+} else alert("This number is not a palindrome");
+
+let number = +prompt("Enter amount of money in US dollars");
+let currency = prompt("EUR = 1, UAH = 2, AZN = 3");
+switch (currency) {
+  case "1":
+    alert(number * 0.89 + "EUR");
+    break;
+  case "2":
+    alert(number * 27.54 + "UAH");
+    break;
+  case "3":
+    alert(number * 1.7 + "AZN");
+    break;
+}
+
+let amount = +prompt("Enter amount of money");
+if (amount >= 200 && amount < 300)
+  alert((amount / 100) * 97 + "$ with 3% discount");
+if (amount >= 300 && amount < 500)
+  alert((amount / 100) * 95 + "$ with 5% discount");
+if (amount >= 500) alert((amount / 100) * 93 + "$ with 7% discount");
+
+let firstValue = +prompt("Enter circumference");
+let secondValue = +prompt("Enter perimeter of a square");
+alert(
+  secondValue / 4 >= firstValue / 3.14
+    ? "Circle fits in a square"
+    : "Circle doesn`t fit in a square"
+);
+
+let firstQuestion = prompt(
+  "Choose the right option:in what year did Napoleon die : 1821 , 1917 , 1656"
+);
+switch (firstQuestion) {
+  case "1821":
+    alert("Hell yeah! You are right.You earned 2 points");

```

Comment:
я бы здесь на месте и добавлял очки
```suggestion
    alert("Hell yeah! You are right.You earned 2 points");
    score += 2;
```